assets/airports/kstl.json

```diff
@@ -0,0 +1,6728 @@
+                ["DSAPP", "ROTRY"]
+            ]
+        },
+        "TEDDD3": {
+            "icao": "TEDDD3",
+            "name": "Ted Three",
+            "rwy": {
+                "6": [],
+                "11": [["FUNKE", "A25+"], "RAMKR"],
+                "12L": [["FUNKE", "A25+"], "RAMKR"],
+                "12R": [["FUNKE", "A25+"], "RAMKR"],
+                "24": [],
+                "29": [["DSAPP", "A25+"], "ANNII", "LENXX", "KAEYE"],
+                "30L": [["WOLTR", "A25+"], "ANNII", "LENXX", "KAEYE"],
+                "30R": [["WOLTR", "A25+"], "ANNII", "LENXX", "KAEYE"]
+            },
+            "body": ["TEDDD"],
+            "exitPoints": {
+                "HILTS": ["ENL", "ZOLLI", "HILTS"],
+                "IIU": ["ENL", "ZOLLI", "HILTS", "IIU"]
+            },
+            "draw": [
+                ["KAEYE", "TEDDD", "ENL", "ZOLLI", "HILTS*"],
+                ["RAMKR", "TEDDD"],
+                ["HILTS", "IIU*"]
+            ]
+        },
+        "TURBO8": {
+            "icao": "TURBO8",
+            "name": "Turbo Eight",
+            "rwy": {
+                "6": [],
+                "11": [],
+                "12L": [],
+                "12R": [],
+                "24": [],
+                "29": [],
+                "30L": [],
+                "30R": []
+            },
+            "body": ["CSX"],
+            "exitPoints": {
+                "AXC": ["FRALE", "WOOBI", "AXC"],
+                "ENL": ["LEEAN", "VANTS", "ENL"],
+                "FAM": ["WEDDG", "FAM"],
+                "GENTS": ["LEEAN", "ZEROJ", "SPORT", "GENTS"],
+                "PXV": ["LEEAN", "ZEROJ", "SPORT", "THOMP", "PXV"],
+                "THOMP": ["LEEAN", "ZEROJ", "SPORT", "THOMP"]
+            },
+            "draw": [
+                ["CSX", "FRALE", "WOOBI", "AXC*"],
+                ["CSX", "LEEAN", "ZEROJ", "SPORT", "THOMP", "PXV*"],
+                ["CSX", "WEDDG", "FAM*"],
+                ["LEEAN", "VANTS", "ENL*"],
+                ["SPORT", "GENTS*"],
+                ["THOMP*"]
+            ]
+        },
+        "WHRLI4": {
+            "icao": "WHRLI4",
+            "name": "Whirly Four",
+            "rwy": {
+                "6": [],
+                "11": [["LUUBE", "A25+"], "DAFUS", "TRAVS", "SEKAY", "LIISA"],
+                "12L": [["HEFNR", "A25+"], "DAFUS", "TRAVS", "SEKAY", "LIISA"],
+                "12R": [["HEFNR", "A25+"], "DAFUS", "TRAVS", "SEKAY", "LIISA"],
+                "24": [],
+                "29": [["HAGEE", "A25+"]],
+                "30L": [["PYUNG", "A25+"]],
+                "30R": [["PYUNG", "A25+"]]
+            },
+            "body": ["WHRLI"],
+            "exitPoints": {
+                "MCM": ["MRCOS", "MCM"]
+            },
+            "draw": [
+                ["LIISA", "WHRLI", "MRCOS", "MCM*"],
+                ["PYUNG", "WHRLI"],
+                ["HAGEE", "WHRLI"]
+            ]
+        }
+    },
+    "stars": {
+        "AARCH1": {
```

Comment:
Please update this and all STARs to include the ranged restrictions posted in the charts.